### PR TITLE
Move header and footer from shared to each domain

### PR DESCRIPTION
**Pull request by muzafarova**

### CODE REVIEW

File: templates/hotel-booking/layout/footer.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { html } from '/src/main.js';
```
Comment (qubyte) [inline templates/hotel-booking/layout/footer.js]: This file is an old version.

File: templates/internal/layout/header.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { html } from '/src/main.js';
```
Comment (qubyte) [inline templates/internal/layout/header.js]: Same with this file.

File: templates/broadband-signup/layout/index.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import summary from './summary.js';
+import header from './header.js';
+import footer from './footer.js';
 import notFound from '../../shared/not-found.js';
```
Comment (qubyte) [inline templates/broadband-signup/layout/index.js]: not-found should probably get the same treatmen.

Comment (muzafarova) [reply templates/broadband-signup/layout/index.js]: I was thinking of error and not-found: both not partials of a layout, but not sections as well. Both are content for the main target (like sections), so in the end, I put error.js' to the root of each domain's:
```
domain
- error.js
- sections
- layout
```

are you ok with such treatment? can do same for not-found

File: templates/hotel-booking/layout/footer.js (hunk 1 above)
Comment (muzafarova) [reply templates/hotel-booking/layout/footer.js]: I blame rebase, I did it in master before checking out to a new branch from master :D will fix

File: templates/broadband-signup/layout/index.js (hunk 3 above)
Comment (qubyte) [reply templates/broadband-signup/layout/index.js]: Sure, that's fine by me.

Comment (muzafarova) [reply templates/broadband-signup/layout/index.js]: ok, will do in this PR

File: templates/internal/layout/header.js (hunk 2 above)
Comment (muzafarova) [reply templates/internal/layout/header.js]: fixed (an in all other domains as well)

File: templates/broadband-signup/layout/index.js (hunk 3 above)
Comment (muzafarova) [reply templates/broadband-signup/layout/index.js]: oh no, actually I'll rather open a separate PR because I've touched error.js in another branch :D

**Pull request merged**